doc/start/hardware-recommendations.rst

```diff
@@ -120,16 +125,16 @@ multiple daemons against a single drive can slow performance considerably.
 Hard Disk Drives
 ----------------
 
-OSDs should have plenty of hard disk drive space for object data. We recommend a
-minimum hard disk drive size of 1 terabyte. Consider the cost-per-gigabyte
-advantage of larger disks. We recommend dividing the price of the hard disk
-drive by the number of gigabytes to arrive at a cost per gigabyte, because
-larger drives may have a significant impact on the cost-per-gigabyte. For
-example, a 1 terabyte hard disk priced at $75.00 has a cost of $0.07 per
-gigabyte (i.e., $75 / 1024 = 0.0732). By contrast, a 3 terabyte hard disk priced
-at $150.00 has a cost of $0.05 per gigabyte (i.e., $150 / 3072 = 0.0488). In the
-foregoing example, using the 1 terabyte disks would generally increase the cost
-per gigabyte by 40%--rendering your cluster substantially less cost efficient.
+OSDs should have plenty of disk drive space for object data. We recommend a
```

Comment:
s/disk//

Reply:
I went with "storage drive" for the sake of consistency. We use "storage drive" in the next non-tip paragraph.